quinn-h3/src/connection.rs

```diff
@@ -104,6 +104,8 @@ impl ConnectionRef {
     }
 }
 
+type ConnectionEnd = bool;
```

Comment:
I don't see this adding much value, since it doesn't provide type safety. Maybe just add some comments to the `drive()` function instead, or instead make it an actual enum? (I've done the latter in work code and I did feel it made the code clearer -- and it should be basically from a performance perspective.)

Reply:
I got the enum named `DriveState::{Closed, Running}`. Thanks!